src/main/flight/pid.c

```diff
@@ -86,127 +86,88 @@ static FAST_RAM_ZERO_INIT bool antiGravityEnabled;
-    );
+                 .pid = {
+                     [PID_ROLL] = DEFAULT_PIDS_ROLL,
+                     [PID_PITCH] = DEFAULT_PIDS_PITCH,
+                     [PID_YAW] = DEFAULT_PIDS_YAW,
+                     [PID_LEVEL_LOW] = {100, 0, 10, 40},
+                     [PID_LEVEL_HIGH] = {35, 0, 1, 0},
+                     [PID_MAG] = {40, 0, 0, 0},
+                 },
+
+                 .dFilter = {
+                     [PID_ROLL] = {2, 100, 250, 50},  // wc, dtermlpf, dtermlpf2, smartSmoothing
+                     [PID_PITCH] = {2, 100, 250, 50}, // wc, dtermlpf, dtermlpf2, smartSmoothing
+                     [PID_YAW] = {0, 100, 250, 0},    // wc, dtermlpf, dtermlpf2, smartSmoothing
+                 },
+
+                 .pidSumLimit = PIDSUM_LIMIT_MAX, .pidSumLimitYaw = PIDSUM_LIMIT_YAW, .dterm_filter_type = FILTER_PT1, .itermWindupPointPercent = 50, .pidAtMinThrottle = PID_STABILISATION_ON, .levelAngleLimit = 45, .angleExpo = 10, .feedForwardTransition = 0, .setPointPTransition[ROLL] = 110, .setPointPTransition[PITCH] = 110, .setPointPTransition[YAW] = 130, .setPointITransition[ROLL] = 75, .setPointITransition[PITCH] = 75, .setPointITransition[YAW] = 45, .setPointDTransition[ROLL] = 110, .setPointDTransition[PITCH] = 110, .setPointDTransition[YAW] = 130, .feathered_pids = 100, .i_decay = 4, .errorBoost = 15, .errorBoostYaw = 40, .errorBoostLimit = 20, .errorBoostLimitYaw = 40, .yawRateAccelLimit = 0, .rateAccelLimit = 0, .itermThrottleThreshold = 350, .itermAcceleratorGain = 1000,
+                 .crash_time = 500,                        // ms
+                 .crash_delay = 0,                         // ms
+                 .crash_recovery_angle = 10,               // degrees
+                 .crash_recovery_rate = 100,               // degrees/second
+                 .crash_dthreshold = 50,                   // degrees/second/second
+                 .crash_gthreshold = 400,                  // degrees/second
+                 .crash_setpoint_threshold = 350,          // degrees/second
+                 .crash_recovery = PID_CRASH_RECOVERY_OFF, // off by default
+                 .horizon_tilt_effect = 130, .nfe_racermode = false, .cinematic_setpoint = false, .crash_limit_yaw = 200, .itermLimit = 400, .throttle_boost = 5, .throttle_boost_cutoff = 15, .iterm_rotation = true, .iterm_relax = ITERM_RELAX_OFF, .iterm_relax_cutoff = 11, .iterm_relax_type = ITERM_RELAX_GYRO, .abs_control_gain = 0, .abs_control_limit = 90, .abs_control_error_limit = 20, .antiGravityMode = ANTI_GRAVITY_SMOOTH, .use_integrated_yaw = false, .integrated_yaw_relax = 200, .motor_output_limit = 100, .auto_profile_cell_count = AUTO_PROFILE_CELL_COUNT_STAY, .horizonTransition = 0, );
 }
 
 void pgResetFn_pidProfiles(pidProfile_t *pidProfiles)
 {
-  for (int i = 0; i < PID_PROFILE_COUNT; i++) {
+    for (int i = 0; i < PID_PROFILE_COUNT; i++)
+    {
         resetPidProfile(&pidProfiles[i]);
     }
 }
 
-extern void init_pwm_filter(float dT);
```

Comment:
why did you remove this?

Reply:
string `init_pwm_filter` is not used anywhere else in the code base.